miso-web/src/main/webapp/styles/images/arrow-back.svg

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="634.975px" height="634.975px" viewBox="0 0 634.975 634.975" style="enable-background:new 0 0 634.975 634.975;"
+	 xml:space="preserve">
+<g>
+	<g id="_x31_">
+		<g>
+			<path d="M283.123,159.091V25.423c0.771-6.686-1.065-13.598-6.232-18.743c-8.975-8.907-23.524-8.907-32.5,0L18.03,254.889
+				c-4.782,4.759-6.822,11.06-6.504,17.292c-0.317,6.232,1.722,12.533,6.504,17.292l225.137,246.849
+				c8.408,7.184,23.819,11.264,33.746,1.359c5.145-5.145,7.388-9.473,6.618-16.158c0,0,0-122.586,0-135.98
+				c149.578,0,284.855,107.695,311.711,249.432c18.085-41.564,28.238-87.344,28.238-135.572
+				C623.48,311.457,471.093,159.091,283.123,159.091z"/>
+		</g>
+	</g>
+</g>

```

Comment:
I think it's weird that all of these SVGs have a bunch of empty groups at the end. I'm not sure why Illustrator does this. Consider deleting them.